src/net/dns.cc

```diff
@@ -149,7 +149,7 @@ class net::dns_resolver::impl
         // do this ourselves, and instead set ares options
         // here, but it seems more error prone (me parsing
         // resolv.conf -> hah!)
-        ares_options a_opts = { 0, };
+        ares_options a_opts = {};
```

Comment:
This a_opts is initialized below. I think it's better to declare it uninitialized, but likely compiler should be smart enough not to _re_-initialize it later

Reply:
yeah, but we don't initialize all its fields. but i think we are obliged to initialize every single bit in this struct. so, probably we should still use `{}` here.

Reply:
Yes, I agree with this place